File: fibonacci/py/code.py
import sys

def fibonacci(n):
  if (n == 0):
    return 0
  if (n == 1):
    return 1
  return fibonacci(n-1) + fibonacci(n-2)

u = int(sys.argv[1])
r = 0
for i in range(u):
  r += fibonacci(i)
print(r)

File: fibonacci/ruby/code.rb
def fibonacci(n)
    return 0 if n == 0
    return 1 if n == 1
    fibonacci(n - 1) + fibonacci(n - 2)
end

u = ARGV[0].to_i
r = 0

(1...u).each do |i|
    r += fibonacci(i)
end

puts r
